csharp/lang/security/crypto/random.cs

```diff
@@ -0,0 +1,6 @@
+var rnd = new Random();
```

Comment:
It looks like this line is missing the `// ruleid:` test annotation. This is probably the cause of the CI failures.